### PR TITLE
maintain script: use a browser to render the page (for vanilla react and such)

### PR DESCRIPTION
**Pull request by SConaway**
In a [thread](https://hackclub.slack.com/archives/C0M8PUPU6/p1617509352275700) about [react-hackclub-webring](https://npmjs.org/react-hackclub-webring) developed by yours truly, @KhushrajRathod mentioned that websites using a non-SSR form of react will fail the checks `scripts/maintain.js` and therefore are unable to be linked to by other sites.

This PR changes `scripts/maintain.js` to render the page in a browser before failing the page (it'll still attempt to search through the unrendered HTML first) so that websites that don't SSR their react still can be on the _directory_. 

It also enables the `bypass` field to be set on any given website which will skip the contents checks altogether (a `bypass`ed website will still have to pass the connectivity check, though). Although I haven't been involved in the webring for too long, I would make sure there is a good reason to allow a website to be `bypass`ed.

### CODE REVIEW

**Comment by googol88** [comment]
Thanks for the PR! The code's looking pretty slick - the react wrapper for the webring is really nice, and as far as I can see the changes in `maintain.js` are all good. Just want to make sure, though, skipping content checks with the `bypass` flag should make websites automatically excluded from `webring/public/members.json`? Because in the case of rishi.cx, there's no webring embed _or_ API so it shouldn't be on the list.

cc @KhushrajRathod and @sampoder then I'd be happy to merge!!

**Comment by SConaway** [comment]
Ok, so I made the `bypass` automatically include them to match how Rishi’s had an exclude before (check the diff—it’s an explicit `if` check).

File: public/members.json [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+[{"member":"Googol","url":"https://googol.now.sh/"},{"member":"Caleb Denio","url":"https://calebdenio.me/"},{"member":"Rafael Cenzano","url":"https://rafaelcenzano.com/"},{"member":"Adrian Lobo","url":"https://loboadrian.github.io/"},{"member":"Hemesh","url":"https://hemesh.tech/"},{"member":"Ong Zhi Zheng","url":"https://ongzz.me/"},{"member":"Anthony Kung","url":"https://anthonykung.com/"},{"member":"Pragya Rajpurohit","url":"https://pragyaraj.tech/"},{"member":"Cole Wilson","url":"https://colewilson.xyz/"},{"member":"Rishi Kothari","url":"https://rishi.cx/","bypass":true},{"member":"Khushraj Rathod","url":"https://www.khushrajrathod.com/"},{"member":"Ali A. Saleh","url":"https://Saleh.HackClub.com/"},{"member":"John Lins","url":"https://www.johnlins.com/"},{"member":"Benjamin Smith","url":"https://benjaminsmith.dev"},{"member":"Tejas Agarwal","url":"https://tejasagarwal.tech/"},{"member":"Vusal Ismayilov","url":"https://woosal.com"}]
```
Comment (sampoder) [inline public/members.json:1]: We need to remove Rishi

Comment (SConaway) [reply public/members.json:1]: Ok, then at that point, I suppose the commit for that can be reverted, or should I keep that?

Comment (sampoder) [reply public/members.json:1]: lets keep support for the bypass but remove it from Rishi

Comment (SConaway) [reply public/members.json:1]: done!

Comment (googol88) [reply public/members.json:1]: Ok, I think it's good now!

**Pull request merged**